Restaurent/Restaurentnew/script.js

```diff
@@ -0,0 +1,523 @@
+let table = [
+  {
+      id: 1,
+      name: "Table 1",  
+      totalItems: Array.from(new Map([])), //[item_id,quantity] new Map([[1,2], [2,3]])
+      cost: 0,
+  },
```

Comment:
This is also okay but try to use the json data from json file where it has only data